### ISSUE TITLE
Please help check my settings for python interpreter. Thanks.

### ISSUE THREAD

**Issue by GuokaiLiu** [shd101wyy/markdown-preview-enhanced#1793]
Here is my code in markdown file.

Here 1 means `

111python {cmd="/mydata/users/liuguo/miniconda2023/envs/xmc/bin/python"}
import os 
os.system('which python')
print(123)
111

Here is the error when I used the code chunk

```
Error: ENOENT: no such file or directory, open '/home/lg/Projects/E3_Reader/f7f3g90cc_code_chunk./mydata/users/liuguo/miniconda2023/envs/xmc/bin/python'
```

**Comment by Liu-Guokai** [comment]
I struggled with this problem for half of the day on both Linux and MacOS.
Finally, I found the solution.

In my case, MPE cannot select/assign the python interpreter.
And it uses the default interpreter when you open your code in the terminal in that environment.

Thus, two steps:
1. Switch to your expected environment,
2. Open VSCode in that terminal.

**Comment by Liu-Guokai** [comment]
I struggled with this problem for half of the day on both Linux and MacOS.
Finally, I found the solution.

In my case, MPE cannot select/assign the python interpreter.
And it uses the default interpreter when you open your code in the terminal in that environment.

Thus, two steps:
1. Switch to your expected environment
2. Open VSCode in that terminal.

See: https://guokliu.blog.csdn.net/article/details/129290524?spm=1001.2014.3001.5502
